github/resource_github_actions_secret.go

```diff
@@ -16,9 +16,8 @@ import (
 
 func resourceGithubActionsSecret() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceGithubActionsSecretCreateOrUpdate,
+		Create: resourceGithubActionsSecretCreate,
```

Comment:
This function does not exist. I submitted another PR, but feel free to update this one and I'll close mine in favour of this.